tests/configuration/test_advanced.py

```diff
@@ -114,3 +114,40 @@ def test_advanced_tot_charge_settings():
     w.reset()
 
     assert w.value.get("total_charge") == 0.0
+
+
+def test_advanced_kpoints_mesh():
+    """Test Mesh Grid HTML widget."""
+    from aiida.plugins import DataFactory
+
+    from aiidalab_qe.app.configuration.advanced import AdvancedSettings
+
+    StructureData = DataFactory("structure")
+
+    w = AdvancedSettings()
+
+    # Create a StructureData for AdvancedSettings (Silicon)
+
+    structure = StructureData(
```

Comment:
```suggestion
    structure = orm.StructureData(
```
We usually have this in other packages.

Reply:
no problems i will call the StructureData via from aiida import orm,